### PR TITLE
docs: fix CourseCard story

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

Updates the storybook examples for the CourseCard to remove imageSrc as a field being passed (since it's not a prop of CourseCard)

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: [ABC-123]
- [X] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Visit CourseCard story in the storybook preview 
](https://github.com/Codecademy/gamut/pull/2677)
Note that none of the examples provide 'imageUrl' as a prop to the CourseCard component

Before:
(see `imageSrc` being passed as a prop, but that image isn't the one that's being rendered on the card. course card has a hardcoded image)
Screen Shot 2023-02-10 at 2 20 34 PM

After:
(no `imageSrc` and an image is still visible :) 
Screen Shot 2023-02-10 at 2 36 20 PM

#### PR Links and Envs

Given this is just a story book update -- I did not create a PRs in mono & monolith to test, but let me know if you would like to me to! 

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |

### CODE REVIEW

File: packages/styleguide/stories/Brand Labs/Molecules/CourseCard.stories.mdx [hunk 1]
```diff
@@ -67,13 +67,7 @@ We only show certificates for pro exclusive courses, since only pro users can do
 
 <Canvas>
   <Story name="CourseCard with Image">
-    {(args) => (
-      <CourseCard
-        {...args}
-        isFullSize
-        imageSrc="https://static-assets.codecademy.com/components/curriculum/path/front-end-engineer-career-path/curriculum-card.svg"
```
Comment (saghdaey) [inline packages/styleguide/stories/Brand Labs/Molecules/CourseCard.stories.mdx:74]: imagesSrc isn't a prop of the CourseCard

Comment (saghdaey) [reply packages/styleguide/stories/Brand Labs/Molecules/CourseCard.stories.mdx:74]: https://github.com/Codecademy/gamut/blob/main/packages/gamut-labs/src/CourseCard/index.tsx#L10

```
export type CourseCardProps = {
  title: string;
  lessonCount: number;
  grantsCertificate: boolean;
  enrollmentStatus: EnrollmentStatus;
  pro: boolean;
  shortDescription?: string | null;
  difficulty?: CourseDifficulty | null;
  isFullSize?: boolean;
  minHeight?: CardProps['minHeight'];
  minWidth?: CardProps['minWidth'];
};
```

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@63.5.6-alpha.49b3ac.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63e699bdccbb6b0aadb19f89--gamut-preview.netlify.app>https://63e699bdccbb6b0aadb19f89--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63e699bdccbb6b0aadb19f89>Deploy Logs</a></p>

**Review by zippyzow — APPROVED** [review]

**Review by dreamwasp — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@63.5.9-alpha.3d2737.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f65c7f4775bf009439d18c--gamut-preview.netlify.app>https://63f65c7f4775bf009439d18c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f65c7f4775bf009439d18c>Deploy Logs</a></p>

**Pull request merged**